packages/kamino-sdk/package.json

```diff
@@ -39,7 +39,7 @@
     "@hubbleprotocol/scope-sdk": "^1.0.279",
     "@orca-so/sdk": "^1.2.25",
     "@orca-so/whirlpool-client-sdk": "^0.0.8",
-    "@orca-so/whirlpool-sdk": "^0.4.2",
+    "@orca-so/whirlpools-sdk": "^0.9.0",
```

Comment:
Are you sure everything works with this change? I think I tried uograding once and found a few unsupported methods in the new SDK, but I could be wrong ofc